Assets/HoloToolkit/Utilities/Scripts/ManualCameraControl.cs

```diff
@@ -28,6 +28,18 @@ public enum MouseButton
     public MouseButton MouseLookButton = MouseButton.Shift;
     public bool IsControllerLookInverted = true;
 
+    public enum ControlMode
```

Comment:
Trying to understand the issue: Is it that the class is not in the HoloToolkit.Unity namespace?